Makefile

```diff
@@ -0,0 +1,6 @@
+OTEL_DOCKER_PROTOBUF ?= otel/build-protobuf:0.4.0
+PROTOC := docker run --rm -u ${shell id -u} -v${PWD}:${PWD} -w${PWD} ${OTEL_DOCKER_PROTOBUF} --proto_path=${PWD}
```

Comment:
I don't have a lot of experience with docker, what does this do? Does it generate the go protobuf bindings?

Reply:
It generates the protobuf files using that docker image instead of requiring to have protoc and goproto plugin.